src/lib/components/tablev2/Tablev2.component.tsx

```diff
@@ -163,6 +166,22 @@ function Table<
     }
   }, [globalFilter, setGlobalFilter, data]);
 
+  const filters = useMemoCompare(
+    allFilters,
+    (previous, next) => JSON.stringify(previous) === JSON.stringify(next),
+  );
+
+  useEffect(() => {
+    if (!filters) {
+      setAllFilters([]);
+    } else {
+      const validFilters = filters.filter(
+        (filter) => filter.id && filter.value,
```

Comment:
I'm unsure about this `&& filter.value` check. Maybe there are some use cases were we would want to filter on falsy values ?